homeassistant/components/modbus/modbus.py

```diff
@@ -103,15 +106,18 @@
 ]
 
 
+_MODBUSHUBS: dict[str, ModbusHub] = {}
```

Comment:
It's better to use hass.data. This is not really a constant so we should avoid using it like that.

Reply:
Ok, I tried to avoid hass.data